cabinets/cabinet/__init__.py

```diff
@@ -38,18 +39,34 @@ def set_configuration(cls, **kwargs):
         pass  # pragma: no cover
 
     @classmethod
-    def read(cls, path, raw=False, **kwargs):
-        if raw:
-            return cls.read_content(path)
-        else:
-            return Parser.load(path, cls.read_content(path), **kwargs)
+    def read(cls, path, parser: Union[bool, Type[Parser]] = True, **kwargs):
+        if parser is True:
+            return Parser.load(path, cls.read_content(path, **kwargs), **kwargs)
+        if parser is False:
+            return cls.read_content(path, **kwargs)
+        try:
+            if issubclass(parser, Parser):
+                return parser.load_content(cls.read_content(path, **kwargs), **kwargs)
+        except TypeError:
+            pass
+
+        raise CabinetError(
+            'Argument `parser` must be True, False or a `Parser` subclass')
```

Comment:
`True` and `False` are values, so I'm wondering if it actually makes sense to wrap them in backticks as well. I don't love how it looks, but it seems maybe more correct?

Reply:
Yep!